tests/playbooks/tests_ethernet.yml

```diff
@@ -60,3 +60,8 @@
       vars:
         state: absent
     - include_tasks: tasks/assert_device_absent.yml
+
+- name: Test warning and info logs
+  hosts: all
+  tasks:
+    - include_tasks: tasks/assert_output_logs.yml
```

Comment:
Please add this to the "Test static interface up" playbook to ensure that the output from there is tested.

Reply:
Ok, thanks! This was one of the things that I was not sure I was making right. In a future test implementation, how should I decide where to add the new task?